src/core/db/models.py

```diff
@@ -71,15 +71,16 @@ class Category(Base):
     """Модель категорий."""
 
     __tablename__ = "categories"
+    __allow_unmapped__ = True
     name: Mapped[str] = mapped_column(String(100))
-    is_archived: Mapped[bool]
+    is_archived: Mapped[bool] = mapped_column(default=False)
 
     users: Mapped[list["User"]] = relationship(secondary="users_categories", back_populates="categories")
 
     tasks: Mapped[list["Task"]] = relationship(back_populates="category")
 
     parent_id: Mapped[int] = mapped_column(Integer, ForeignKey("categories.id"), nullable=True)
-    children = relationship("Category", backref=backref("parent", remote_side="Category.id"))
+    children: relationship("Category", backref=backref("parent", remote_side="Category.id"))
```

Comment:
Это не должно работать...

Reply:
Это так и было. Убрать или что с этим делать?

Reply:
Не похоже что это так и было. [Посмотри в документации синтаксис](https://docs.sqlalchemy.org/en/20/orm/basic_relationships.html)
```suggestion
    children = relationship("Category", backref=backref("parent", remote_side="Category.id"))
```
Можно добавить аннотации:
```suggestion
    children: Mapped["Category"] = relationship("Category", backref=backref("parent", remote_side="Category.id"))
```

Reply:
точно,  это я исправил. Приношу извинения, я видно увлекся исправлением других полей. Сейчас все сделаю